geode-core/src/main/java/org/apache/geode/Delta.java

```diff
@@ -55,7 +55,20 @@
   void fromDelta(DataInput in) throws IOException, InvalidDeltaException;
 
   /**
-   * Allows Delta implementations to ensure bucket sizes are recalculated after delta is applied
+   * By default, buckets do not recalculate their size when deltas are applied. This optimizes for
```

Comment:
Instead of "buckets" I would focus this on deltas. So say something like: "By default, the size of an entry value that implements the Delta interface is not recalculated when the delta changes.